packages/components/package.json

```diff
@@ -75,17 +75,18 @@
 		"@storybook/react": "6.2.9",
 		"@testing-library/react": "11.2.7",
 		"@testing-library/user-event": "13.2.1",
+		"@woocommerce/style-build": "file:../style-build",
 		"@woocommerce/wc-admin-settings": "file:../wc-admin-settings",
 		"@wordpress/scripts": "12.6.1",
 		"concurrently": "5.3.0"
 	},
 	"scripts": {
 		"build": "npm run build:js && npm run build:css",
 		"build:js": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
-		"build:css": "node ../../bin/packages/build.js src/style.scss",
-		"clean": "npx rimraf build build-*",
+		"build:css": "webpack",
+		"clean": "npx rimraf tsconfig.tsbuildinfo build build-*",
 		"prepack": "npm run clean && npm run build",
-		"start": "concurrently \"tsc --build --watch\" \"node ../../bin/packages/watch.js ./src\"",
+		"start": "../../node_modules/.bin/concurrently \"tsc --build --watch\" \"webpack --watch\"",
```

Comment:
Does declaring `concurrently` as a devDependency no longer work? It would be nice to avoid calling `concurrently` from the root node_modules

Reply:
It doesn't work unless you install node modules in each package directory. I forgot that when testing `npm start` in each directory - from the [initial PR](https://github.com/woocommerce/woocommerce-admin/pull/7286):

> Test running scripts for a single package (`npm run start:package @woocommerce/<package>`)

So I'll revert the change and single package watch scripts will need to be run from the root using the `start:package` command (for now - which isn't part of the dev workflow as of yet).